src/main/resources/application.properties

```diff
@@ -46,6 +46,7 @@ datarepo.firestoreSnapshotBatchSize=500
 datarepo.firestoreSnapshotCacheSize=200
 datarepo.firestoreValidateBatchSize=500
 datarepo.firestoreQueryBatchSize=10000
+datarepo.authCacheTimeoutSeconds=60
```

Comment:
You will want to put the cache size here, too. Right?
And you will need to add them to `bio/terra/app/configuration/ApplicationConfiguration.java` with the appropriate accessors.